### DISCUSSION TITLE
After every step add a new step-readme file instead of replacing the main readme file

### DISCUSSION THREAD

**Discussion by ozkary** [skills/.github#301] [Code with Copilot]
As we make progress for each step, it will be better to add a new md file for each new step created. The main readme file could get updated with the new step link to signal to the users to know click that new file for the next step.

This should allow readers to browse back previous files and understand how new step readme files are being added via the GitHub actions. 

readme.md
step1-readme.md
step2-readme.md
...

Thanks.

**Comment by vitamin2025** [comment]
If you want to browse back, there is actually a way for that.

The bot is not writing the instructions from scratch rather, it's just copy pasting pre-existing files that are already there in your repo.

You can view these pre existing instructions in your repo in this folder: .github > steps

![image](https://github.com/skills/.github/assets/72799488/5207e255-c279-4e31-b5d0-90e2bddfa723)
